tokens/src/core/global/display.json

```diff
@@ -2,19 +2,19 @@
   "typography": {
     "display": {
       "line-height": {
-        "base": { "value": "1", "type": "dimension", "source": "$display-line-height" },
-        "mobile": { "value": "3.5rem", "type": "dimension", "source": "$display-mobile-line-height" }
+        "base": { "value": "1", "type": "dimension", "source": "$display-line-height", "description": "Line height basic value." },
+        "mobile": { "value": "3.5rem", "type": "dimension", "source": "$display-mobile-line-height", "description": "Line height for mobile devices." }
       },
-      "1": { "value": "3.75rem", "type": "dimension", "source": "$display1-size" },
-      "2": { "value": "4.875rem", "type": "dimension", "source": "$display2-size" },
-      "3": { "value": "5.625rem", "type": "dimension", "source": "$display3-size" },
-      "4": { "value": "7.5rem", "type": "dimension", "source": "$display4-size" },
-      "mobile": { "value": "3.25rem", "type": "dimension", "source": "$display-mobile-size" },
+      "1": { "value": "3.75rem", "type": "dimension", "source": "$display1-size", "description": "Font size of level 1." },
+      "2": { "value": "4.875rem", "type": "dimension", "source": "$display2-size", "description": "Font size of level 2." },
+      "3": { "value": "5.625rem", "type": "dimension", "source": "$display3-size", "description": "Font size of level 3." },
+      "4": { "value": "7.5rem", "type": "dimension", "source": "$display4-size", "description": "Font size of level 4." },
```

Comment:
"level 1" etc. doesn't seem to add any context here. Not sure what we should have here instead though.

Reply:
I think here would be a good idea to look where we use these variables and come up with more detailed description. I think these are font sizes for headers? Maybe the description would be something like `Base font size for h1 element.` (not ideal but probably better than saying `level 1`)

Reply:
Changed to "Font size for heading of level 1." and so on